roles/keystone-setup/tasks/main.yml

```diff
@@ -97,3 +97,7 @@
 
 - name: restart keystone api
   service: name=keystone state=restarted
+
+- include: federation.yml
+  when: keystone.federation.enabled|bool
+  tags: [ 'keystone-federation']
```

Comment:
this is a tag, so it is possible to run at-will today...without running all of keystone-setup